Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -14,6 +14,7 @@ param(
     [switch] $quiet
 )
 Import-Module -Name .\Install-Identity-Utilities.psm1 -Force
+Import-Module -Name .\Install-IdPSS.psm1 -Force
```

Comment:
Wrong script name, but will update when moving the Alter-Config script into here